### DISCUSSION TITLE
Feature Requests

### DISCUSSION THREAD

**Discussion by bdejesus** [bdejesus/xiv-bars#204] [Ideas]
Discuss feature requests here...

**Comment by jacobohpea** [comment]
As a user
I want to sort the job specific lists by likes
so that I can find the most liked and assumed highest quality layouts

Sort button shows on job page
Sort button allows for sorting by likes and by preferred layout

**Reply by bdejesus** [reply]
👋🏼 Thanks for the suggestion! I've added filtering to the Job page, it's pretty basic, but should work for now.
